uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -539,10 +602,12 @@ define([
        */
       this._init = function () {
         var self = this;
+        vm = this;
```

Comment:
@AkA84 is it ok to leave the *vm* var here or should I refactor? I'm using it for convenience saque somewhere else in the code.